photon-api/src/main/scala/com/linkedin/photon/ml/optimization/GeneralizedLinearOptimizationProblem.scala

```diff
@@ -67,22 +67,6 @@ protected[ml] abstract class GeneralizedLinearOptimizationProblem[Objective <: O
   protected def createModel(coefficients: Vector[Double], variances: Option[Vector[Double]]): GeneralizedLinearModel =
     glmConstructor(Coefficients(coefficients, variances))
 
-  /**
-   * Create a GLM from given normalized coefficients (potentially including intercept)
-   *
-   * @param normalizationContext The normalization context
-   * @param coefficients The feature coefficients means
-   * @param variances The feature coefficient variances
-   * @return A GLM with the provided feature coefficients
-   */
-  protected def createModel(
```

Comment:
`createModel` function in line 67 is the function we are using (does not take `normalizationContext`). We don't use this function because variance has the wrong transformation. `createModel` function is used when we have coefficients and variances in the original space so we don't need to do any transformation inside. See line 131 in `DistributedOptimizationProblem.scala`